packages/core/state/selection/actions.ts

```diff
@@ -515,6 +515,60 @@ export function toggleFileFolderCollapse(fileFolder: FileFolder): ToggleFileFold
     };
 }
 
+/**
+ * COLLAPSE_ALL_FILE_FOLDERS
+ * Intention to collapse all file folders
+ */
+export const COLLAPSE_ALL_FILE_FOLDERS = makeConstant(
+    STATE_BRANCH_NAME,
+    "collapse-all-file-folders"
+);
+
+export interface CollapseAllFileFoldersAction {
+    type: string;
+}
+
+export function collapseAllFileFolders(): CollapseAllFileFoldersAction {
+    return {
+        type: COLLAPSE_ALL_FILE_FOLDERS,
+    };
+}
+
+/**
+ * EXPAND_ALL_FILE_FOLDERS
+ * Intention to toggle the given file folder's collapsed state
+ */
+export const EXPAND_ALL_FILE_FOLDERS = makeConstant(STATE_BRANCH_NAME, "expand-all-file-folders");
```

Comment:
nit: should this match the format/indentation of line 522?

Reply:
I believe this was done by our linter; e.g., it only performs the line break/indentation when we pass some column width